### ISSUE TITLE
Warning: Failed prop type: Invalid prop `style` of type `number` supplied to `UserAvatar`, expected `object`

### ISSUE THREAD

**Issue by rahul-budhia** [avishayil/react-native-user-avatar#52]
I encountered this while trying to pass style object created by `StyleSheet.create()` to `UserAvatar`. It show this warning because of optimisation that `StyleSheet.create` does, where it returns number instead of object

```
index.js:1 Warning: Failed prop type: Invalid prop `style` of type `number` supplied to `UserAvatar`, expected `object`.
    in UserAvatar (at User.js:13)
    in Avatar (at User.js:34)
```

Should be fixed by using `ViewPropTypes` from `react-native` or some module like [react-style-proptype](https://github.com/brigand/react-style-proptype)

**Comment by avishayil** [comment]
Thanks @rahul-budhia for catching this. Can you submit a PR?

**Comment by avishayil** [comment]
Should be fixed with [v1.0.8](https://github.com/avishayil/react-native-user-avatar/releases/tag/v1.0.8)
